parsers/ENTSOE.py

```diff
@@ -1000,6 +1018,67 @@ def parse_prices(
     return prices
 
 
+def get_exchange(
+    zone_key1: ZoneKey,
+    zone_key2: ZoneKey,
+    session: Session | None = None,
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+    forecasted: bool = False,
+) -> list:
+    """Gets either the forcasted or measured net exchange between two zones."""
+    session = session or Session()
+
+    sorted_zone_keys = sorted([zone_key1, zone_key2])
+    key = ZoneKey("->".join(sorted_zone_keys))
+    if key in ENTSOE_EXCHANGE_DOMAIN_OVERRIDE:
+        domain1, domain2 = ENTSOE_EXCHANGE_DOMAIN_OVERRIDE[key]
+    else:
+        domain1 = ENTSOE_DOMAIN_MAPPINGS[zone_key1]
+        domain2 = ENTSOE_DOMAIN_MAPPINGS[zone_key2]
+    # Grab exchange
+    # Export
+    raw_exchange = (
+        query_exchange_forecast(domain1, domain2, session, target_datetime)
+        if forecasted
+        else query_exchange(domain1, domain2, session, target_datetime)
+    )
+    if raw_exchange is not None:
+        parsed = parse_exchange(
+            raw_exchange,
+            is_import=False,
+            zoneKey1=zone_key1,
+            zoneKey2=zone_key2,
+            logger=logger,
+            forecasted=forecasted,
+        )
+        if parsed:
```

Comment:
Are you sure about this ?
Shouldn't we try to get exports and imports independently of whether there's an import or export ?

Reply:
As far as I am aware they are always matched and since we parse them alphabetically the order will always be the same. But I can look into making this more robust.